src/Compilers/Test/Utilities/CSharp/UsesIsNullableVisitor.cs

```diff
@@ -48,6 +48,11 @@ public override bool VisitMethod(MethodSymbol symbol)
                 VisitList(symbol.Parameters);
         }
 
+        public override bool VisitParameter(ParameterSymbol symbol)
+        {
+            return AddIfUsesIsNullable(symbol.ContainingSymbol, symbol.Type);
+        }
```

Comment:
Parameters should be handled as part of the `ContainingSymbol`. See `VisitMethod` for instance.

Reply:
Actually, `VisitMethod` looks incorrect after all. It feels like there should be a `VisitList` overload that takes a `containingSymbol` argument and that `containingSymbol` should be added to `_builder` if any of the symbols in the list use `IsNullable`. `VisitMethod` could call `|| VisitList(symbol, symbol.Parameters)`.